Data/ByteString/Lazy/Char8.hs

```diff
@@ -84,9 +86,9 @@ module Data.ByteString.Lazy.Char8 (
         -- * Building ByteStrings
         -- ** Scans
         scanl,
---      scanl1,
---      scanr,
---      scanr1,
+     scanl1,
+     scanr,
+     scanr1,
```

Comment:
Align with other entries please.

Reply:
Sure. I have poor vision on indentation so I often err by a handful of spaces.